aws-lambda/src/main.rs

```diff
@@ -13,8 +15,17 @@ mod runtime;
 
 #[tokio::main]
 async fn main() -> Result<(), Error> {
+    let _ = dotenv();
+
+    let trace: tracing::Level = std::env::var("LOG_LEVEL")
+        .ok()
+        .as_ref()
+        .and_then(|x| tracing::Level::from_str(x).ok())
+        // log everything by default since logs can be filtered by level in CloudWatch.
+        .unwrap_or(tracing::Level::TRACE);
```

Comment:
```suggestion
        .unwrap_or(tracing::Level::INFO);
```
to be consistent with our documentation

Reply:
Read the comment above.

Reply:
And [this](https://github.com/tailcallhq/tailcall/pull/1171#discussion_r1484304309).